scripts/build-upload-docker-images.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+set -euxf -o pipefail
+
+build_upload_multiarch_images(){
+  for component in agent collector query ingester
+  do
+		docker buildx build --output "${PUSHTAG}" \
+    	--progress=plain --target release \
+    	--build-arg base_image="localhost:5000/baseimg:1.0.0-alpine-3.12" \
+			--build-arg debug_image="golang:1.15-alpine" \
```

Comment:
Yes, you're right, we've noticed this. But building `debugimg:1.0.0-golang-1.15-alpine` needs **`go-delve`** which does not have **`s390x`** support yet. So **at this moment** we only make **release version** of docker images (like `all-in-one`, `jaeger-agent`, `jaeger-tracegen`, etc.) **multi-arch** and using `debug_image="golang:1.15-alpine"` in `build-arg`. In fact, the argument `debug_image` is not used when building release version of images.

The **debug version** of images (like `all-in-one-debug` or `jaeger-agent-debug` ) still use `debug_image=localhost/debugimg:1.0.0-golang-1.15-alpine` and are **`amd64` specific**.

Reply:
> In fact, the argument debug_image is not used when building release version of images.

If this argument is not used, is it mandatory to pass it ? It just creates confusion, isn't it ?

Reply:
Yes, since the debug images also use the same Dockerfile, we need to **keep** this arg and **pass** it. Although it would create confusion at this moment, once `go-delve` is supported by `s390x`, we can use this arg for building multi-arch debug images. Otherwise we need to separate each of these Dockerfiles into two Dockerfiles, one for `release` target, the other for `debug` target.